DigitalLearningSolutions.Data/Helpers/LinkedFieldHelper.cs

```diff
@@ -1,6 +1,7 @@
 ﻿namespace DigitalLearningSolutions.Data.Helpers
 {
     using System.Collections.Generic;
+    using System.Linq;

```

Comment:
Note that on uar-test, this file has been moved to `DigitalLearningSolutions.Web/Helpers/LinkedFieldHelper.cs`, so you may get merge conflicts